src/dmd/target.d

```diff
@@ -783,26 +785,25 @@ extern (C++) struct Target
         }
         tn = tn.toBasetype();
         //printf("tn = %s\n", tn.toChars());
-        d_uns64 sz = tn.size();
+        auto sz = tn.size();
         Type tns = tn;
 
         if (os == Target.OS.Windows && is64bit)
         {
             // http://msdn.microsoft.com/en-us/library/7572ztz4.aspx
-            if (tns.ty == Tcomplex32)
+            if (tns.ty == TY.Tcomplex32)
                 return true;
             if (tns.isscalar())
                 return false;
 
             tns = tns.baseElemOf();
-            if (tns.ty == Tstruct)
+            if (auto ts = tns.isTypeStruct())
             {
-                StructDeclaration sd = (cast(TypeStruct)tns).sym;
```

Comment:
The diff would have been smaller if this line became: `StructDeclaration sd = ts.sym`

Reply:
`StructDeclaration` needs importing though.

Reply:
`auto` ?

Reply:
I don't see it worth it for a two line saving.